core/new-gui/src/app/workspace/component/product-tour/product-tour.component.ts

```diff
@@ -1,38 +1,162 @@
 import { Component, OnInit } from '@angular/core';
 import { TourService, IStepOption } from 'ngx-tour-ng-bootstrap';
 
+/**
+ * ProductTourComponent is the product tour that shows basic product tutorial.
+ *
+ * Product tour library built with Angular (2+).
+ * ngx-tour-ngx-bootstrap is an implementation of the tour ui that uses ngx-bootstrap popovers to display tour steps.
+ *
+ * The component has a step list in this.tourService.initialize that can add, edit or delete steps.
+ * Define anchor points for the tour steps by adding the tourAnchor directive throughout components.
+ *
+ * <div tourAnchor="some.anchor.id">...</div>
+ *
+ * Define your tour steps using tourService.initialize(steps).
+ *
+ * For the full text of the library, go to https://github.com/isaacplmann/ngx-tour
+ *
+ * The screenshots were done by GIPHY Capture
+ *
+ *
+ * @author Bolin Chen
+ */
+
 @Component({
   selector: 'texera-product-tour',
   templateUrl: './product-tour.component.html',
   styleUrls: ['./product-tour.component.scss']
 })
 export class ProductTourComponent implements OnInit {
 
-  public loaded = false;
+  private steps: IStepOption[] = [{
+    anchorId: 'texera-navigation-grid-container',
+    content: `
+    <div class="intro">
+    <center>
+      <h3>Welcome to Texera!</h3>
+    </center>
+    <br>
+    <p>
+    Texera is a system to support cloud-based text analytics using declarative and GUI-based workflows. Use <b>« Prev</b> and
+     <b>Next »</b> or left and right arrow keys to navigate through the tutorial.
+    </p>
+    <br>
+    <center>
+    <img src="../../../assets/Tutor_Intro_Sample.jpeg" alt="intro img">
+    </center>
+    <br><br>
+    </div>
+    `,
+    placement: 'bottom',
+    title: 'Welcome',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-operator-panel',
+    content: `
+    <p>This <b>Operator Panel</b> contains all the operators we need. </p>
```

Comment:
<p>This <b>Operator Panel</b> contains all the operators you need to construct a workflow. </p> 
 <p>Let's create a basic twitter text analysis workflow to demonstrate the steps involved in creation of a workflow. <p>Open the first section named <b>Source</b>.</p>